### ISSUE TITLE
Playlist controls not showing

### ISSUE THREAD

**Issue by Lagg** [ParticleCore/Particle#272]
Hi, I left a comment on the extension page but this seems like a better avenue. I'll put the contents of that comment here:

The div.control-bar that has your stuff along with the other playlist controls doesn't appear. In fact I feel like this may be an upstream issue.

I wrote a GM script that simply moves div.control-bar to under .main-content, as a sibling of .playlist-header. Simple fix and doesn't appear to affect functionality. Great extension otherwise.

**Comment by Lagg** [comment]
In fact here's the script in its entirety:

`$(".playlist-header").after($(".control-bar").detach());`

**Comment by Lagg** [comment]
**Edit**: I decided to not be silly and remembered to test correctly with incognito. Seems like this isn't upstream. I can see the controls that are there otherwise for shuffle and stuff. I don't have any userscripts running besides what was mentioned above. I'm not really seeing where this can be happening styling wise. At least in the poking around I could do in the dev tools.

**Comment by ParticleCore** [comment]
I tested this in Firefox and Chrome, they both show the playlist controls as they should. If it works in Incognito then it is very likely this is on your end only, possibly a YouTube layout test. Clearing the cache and logging out/in should be sufficient to force a new cookie to be sent, otherwise I'd suggest that you follow the troubleshooting guide since I am not able to replicate this on my end: https://github.com/ParticleCore/Particle/wiki/Report-a-problem

**Comment by Lagg** [comment]
I attempted a clear & relaunch and turned off anything that could be affecting content (like ublock) except for jsonview. Any other ideas to see what could be causing it?

**Comment by ParticleCore** [comment]
Have you followed the troubleshooting guide? Because it contains almost everything I would ask you to try. Including additional information would also be very helpful, which is explained in the guide, namely what operating system, browser and YT+ version and type you are using, additionally a screenshot showing the problem with YT+ turned on and turned off could be very helpful. Otherwise I will have to repeat everything that is already explained in the guide.

Turning off just the extensions and userscripts that you think might be affecting is not enough, anything can be affecting which is why it is best to disable everything except YT+, restart and see if the problem persists, otherwise we can never confirm whether this is being caused by a conflict with another extension.

**Comment by Lagg** [comment]
Yeah though one of the things you suggest in there - creating a new profile - isn't something I can really do (your stuff is working at like 98% capacity, but I need that 2% as well for this specific usecase). That's about the only thing I didn't try. I'll attempt  to disable hangouts, jsonview, etc. And oops, apologies for not including info. I actually missed that section entirely, I guess because I ctrl-w'd after going through the prior steps.

```
Browser Name: Chrome 

Browser Version: 51.0.2704.84 m

Operating System: Windows 7

Software Type: Extension

Software Version: 1.3.0

Userscript Manager: Tampermonkey

Userscript Version: N/A

Problem Description: Above

Steps to reproduce: Click play all on use uploads, click video with list param in link qs

```

**Comment by Lagg** [comment]
Yeah now all extensions including jsonview and google crap are off. Absolutely everything *but* YouTube Plus.
![asdf](https://cloud.githubusercontent.com/assets/2509793/16053191/ee619ed2-3224-11e6-8466-02448d98dd59.png)

**Comment by ParticleCore** [comment]
Testing with the same setup as yours and I still cannot replicate the problem

![ss](https://cloud.githubusercontent.com/assets/9222661/16054463/cc1e3eec-325c-11e6-827b-ba6901538e9f.png)


If you haven't reset the YT+ settings to their default please do so, restart the browser and try again. Remember to backup your settings in case you have been using the blacklist feature.

If you already have tried the previous step then try the extension version to see if this is restricted to the userscript version:

https://chrome.google.com/webstore/detail/youtube-plus/nkdalpbojfdilmiboaiedicdbigdabpb

**Comment by Lagg** [comment]
Alright, I wanted to have a provably fresh restart/re-enable/reset so I took the screenshot as is. Sorry for it being different from previous player modes. Will amend this post after trying another version. Going to just try the userscript. I can easily see some kind of weirdness in chrome's APIs doing this.

![untitled](https://cloud.githubusercontent.com/assets/2509793/16055101/33ff8a32-322d-11e6-8065-407a307db8e0.png)

**Comment by ParticleCore** [comment]
I can already see the top YouTube bar is different which means you received a test cookie causing YouTube to display a different layout. Are you certain you cleared all your cache instead of just a small clean-up spanning a short period?

Trying in a new test profile just with YT+ installed would rule out any possible conflict with extensions, browser settings and cookies. If I cannot replicate it then it is almost impossible for me to fix the problem.

**Comment by Lagg** [comment]
Hrm, seriously? I actually not only did that but also did an incognito run. Did they opt me into something silly here? And though it's pretty hard for me to do profiles right now I'll try to give it a shot anyway because again this thing is /almost/ working perfectly for me. Would like to fix.

**Comment by Lagg** [comment]
Yep, that was the case. It works after I log out (as in explicitly logged out, I'm wondering if this is IP tied) and then log in again. In incognito. I don't know why it's so sticky. Or maybe the razor applies and i selected 1 hour instead of all time on accident in the clear dialog. I would like to consider this issue closed but what if it shows up again? It's pretty hard (and hacky) to write layout-specific code for this unless Google puts an identifier in one of the tags or something.

**Comment by ParticleCore** [comment]
There is no worries, now that I know this issue is tied with that test cookie I will work on a fix for it once I receive it. Until then I simply cannot see what is causing the problem so it will have to wait.

**Comment by Lagg** [comment]
Alright, fair enough. I wanted to note that this might be tied to the .google.com cookies. Particularly for G+. I'm seeing one called `GPLUS_UI_BETA=1` in  mine. This could be how they were making it so sticky for me. Maybe you can find some testable stuff with that. Thanks for the help.

**Comment by ParticleCore** [comment]
Thank you for the extra info. I suspect that your top layout might be tied with their material UI coming up on beta: http://www.androidpolice.com/2016/04/30/youtube-is-testing-a-new-material-ui-on-the-web/

It should be a matter of time until I get hit with that test cookie as well.

**Comment by Lagg** [comment]
Okay, last note. Definitely be careful when you do get the opportunity to test this. Even though it's gone when i'm signed out, while logged in it repeatedly sets it back to the new layout. So I guess broken controls and probably other stuff until I can find an explicit opt-out. And knowing google - heh.

**Comment by Lagg** [comment]
@ParticleCore Google made VISITOR_INFO1_LIVE HttpOnly. No `document.cookie`. No extension based modification. Unbelievable. Now I can't force it to the same one I see not logged in. Pretty sure this was intentional on their part. My sincere apologies for this issue when it turns out that there's nothing you can possibly do to fix it, and there is also nothing I can do to use one of the most awesome features of your stuff without writing userscript hacks. It truly sucks because this can make my upload flow so much more efficient D':

**Comment by Eisys** [comment]
Not that I can be of any help but I am also having this issue.
But even with YT+ disabled.

The playlist controls only show up in incognito mode.

In normal chrome, I have the new top bar:
http://i.imgur.com/3Gsj28z.png

In incognito, it still has the old one:
http://i.imgur.com/q9qgFhZ.png

**Comment by ParticleCore** [comment]
The fact that @Eisys is having this problem even with the extensions disabled tells me that it is either a YouTube layout bug or they intend to drop those controls from the playlists. Either way, with this confirmation I would suggest that you let YouTube know of this problem.

At the end of the YouTube pages there is a "Send Feedback" button, use it to report this problem, you can even highlight the area of interest, if it is a bug it will be corrected in a months time at best.

**Comment by Lagg** [comment]
If this turns out to be an upstream bug I'm just going to laugh my ass off.
